lib/src/io/node.dart

```diff
@@ -187,30 +191,28 @@ T _systemErrorToFileSystemException<T>(T callback()) {
   }
 }
 
-final stderr = Stderr(process.stderr);
+final stderr = Stderr(process?.stderr ?? createWritable(WritableOptions()));
 
 /// We can't use [process.stdout.isTTY] from `node_interop` because of
 /// pulyaevskiy/node-interop#93: it declares `isTTY` as always non-nullably
 /// available, but in practice it's undefined if stdout isn't a TTY.
-@JS('process.stdout.isTTY')
-external bool? get isTTY;
-
-bool get hasTerminal => isTTY == true;
+// ignore: invalid_null_aware_operator
```

Comment:
This is to work around the issue described in the comment above this line. Without the `ignore`, Dart Analyzer complains about `stdout?.isTTY` because it assumes `isTTY` is always available

Reply:
See the Map class in https://pub.dev/packages/js

Reply:
Shouldn't need this anymore I think

Reply:
Removed in https://github.com/oddbird/dart-sass/commit/1937affd6fa9fc3965c72222f323f56fb53352d7